tests/src/packages/websocket/WebSocketTests.scala

```diff
@@ -55,25 +55,39 @@ class WebSocketTests
     }
 
     it should "Stand up a websocket server for testing" in {
-        println(s"Server URI: ${serverURI}")
-
-        retry({
-            val testClient = wsUtils.getSocketClient(serverURI)
-            val serverStarted = testClient.connectBlocking()
-
-            serverStarted should be(true)
-            testClient.closeBlocking()
-        }, 3, waitBeforeRetry = Some(1 second))
+        WhiskProperties.getProperty("cli.api.host") match {
```

Comment:
looks good. however, we should think how to have test coverage for bluemix deployments.